### PR TITLE
bug(button): properly align text in fab buttons #5632

### PR DESCRIPTION
**Pull request by amcdnl**
Addresses: Link Fab Buttons incorrect line height #5632

**BEFORE**
img2

**AFTER**
img

### CODE REVIEW

File: src/lib/button/_button-theme.scss [hunk 1]
```diff
@@ -107,7 +107,7 @@
 }
 
 @mixin mat-button-typography($config) {
-  .mat-button, .mat-raised-button, .mat-icon-button {
+  .mat-button, .mat-raised-button, .mat-icon-button, .mat-fab, .mat-mini-fab {
```
Comment (jelbourn) [inline src/lib/button/_button-theme.scss]: @crisbeto did you have a reason for omitting the fab buttons here?

Comment (crisbeto) [inline src/lib/button/_button-theme.scss]: I don't remember, but it was probably because the fabs aren't really meant to be used with text.

Comment (amcdnl) [inline src/lib/button/_button-theme.scss]: @crisbeto  Is there any reason why not to?

Comment (crisbeto) [inline src/lib/button/_button-theme.scss]: The spec doesn't really have fabs with text, if I remember correctly, but it shouldn't be an issue to keep it anyway.

Comment (amcdnl) [inline src/lib/button/_button-theme.scss]: @crisbeto  We good to merge then?

**Comment by jelbourn** [comment]
Caretaker note: let me know if this causes any screenshot diffs

**Comment by kara** [comment]
@jelbourn No screenshot diff failures, so merging.

**Pull request merged**